paz/processors/pose.py

```diff
@@ -127,3 +132,249 @@ def call(self, boxes2D):
             y_center = (z_center * v) / self.focal_length
             hands_center.append([x_center, y_center, z_center])
         return np.array(hands_center)
+
+
+class ComputeCameraParameter(Processor):
```

Comment:
This name is misleading. It seems a rearrangment of the camera matrix?

Reply:
This processor is now removed because its only re-arrangement of camera_matrix and addition of image_scale and translation_scale_norm. These values are directly passed and used now wherever required instead of accessing from camera_parameter